notifications/tests/test_notifications.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+
+from notifications.enums import NotificationType
+from notifications.models import NotificationTemplate, NotificationTemplateException
+from notifications.utils import render_notification_template
+
+
+@pytest.fixture
+def notification_template(settings):
+    settings.LANGUAGES = (('fi', 'Finnish'), ('en', 'English'))
+
+    template = NotificationTemplate.objects.language('en').create(
+        type=NotificationType.SIGNUP_CREATED,
```

Comment:
Hmm, notifications in these tests seem to have a weird type... :)

Reply:
😄 oopsie-poopsie 😄